xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrTranslations.xml

```diff
@@ -143,7 +143,7 @@ solr.facet.size.any=Any size
 solr.field.type=Result type
 solr.field.locale=Language
 solr.field.locale.emptyValue=No Language
-solr.field.locale.emptyValue.hint=Some documents don't have a language set because they contain scripts (which are not written in a natural language) or they make up the user interface which is displayed in the current language (like this page).
+solr.field.locale.emptyValue.hint=Some documents don't have a language set because they contain scripts which are not written in a natural language or they make up the user interface which is displayed in the current language (like this page).
```

Comment:
Please put back the parentheses, since it's a mention about what scripts *are*. Without the parentheses it would be interpreted as a mention of a type of script, implying that scripts could be written in natural language, which is false.

Reply:
Done.